css/README.md

```diff
@@ -1,364 +1,350 @@
-   Grid layout
-   ========================================================================== */
-
-/**
- * Column layout with horizontal scroll.
- *
- * This creates a single row of full-height, non-wrapping columns that can
- * be browsed horizontally within their parent.
- *
- * Example HTML:
- *
- * <div class="grid">
- *     <div class="cell cell-3"></div>
- *     <div class="cell cell-3"></div>
- *     <div class="cell cell-3"></div>
- * </div>
- */
-
-/**
- * Grid container
- *
- * Must only contain `.cell` children.
- *
- * 1. Remove inter-cell whitespace
- * 2. Prevent inline-block cells wrapping
- */
-
-.grid {
-    height: 100%;
-    font-size: 0; /* 1 */
-    white-space: nowrap; /* 2 */
-}
+        @media (...) {
+            // ...
+        }
 
-/**
- * Grid cells
- *
- * No explicit width by default. Extend with `.cell-n` classes.
- *
- * 1. Set the inter-cell spacing
- * 2. Reset white-space inherited from `.grid`
- * 3. Reset font-size inherited from `.grid`
- */
-
-.cell {
-    position: relative;
-    display: inline-block;
-    overflow: hidden;
-    box-sizing: border-box;
-    height: 100%;
-    padding: 0 10px; /* 1 */
-    border: 2px solid #333;
-    vertical-align: top;
-    white-space: normal; /* 2 */
-    font-size: 16px; /* 3 */
-}
+        > .icon {
+            margin-right: 10px;
+        }
+    }
+    ```
 
-/* Cell states */
+### Variables
 
-.cell.is-animating {
-    background-color: #fffdec;
-}
+Prefer dash-cased variable names (e.g. `$my-variable`) over camelCased or snake_cased variable names. It is acceptable to prefix variable names that are intended to be used only within the same file with an underscore (e.g. `$_my-variable`).
+
+### Mixins
+
+Mixins should be used to DRY up your code, add clarity, or abstract complexity – in much the same way as well-named functions. Mixins that accept no arguments can be useful for this, but note that if you are not compressing your payload (e.g. gzip), this may contribute to unnecessary code duplication in the resulting styles.
 
-/* Cell dimensions
-   ========================================================================== */
+### Extend directive
 
-.cell-1 { width: 10%; }
-.cell-2 { width: 20%; }
-.cell-3 { width: 30%; }
-.cell-4 { width: 40%; }
-.cell-5 { width: 50%; }
+`@extend` should be avoided because it has unintuitive and potentially dangerous behavior, especially when used with nested selectors. Even extending top-level placeholder selectors can cause problems if the order of selectors ends up changing later (e.g. if they are in other files and the order the files are loaded shifts). Gzipping should handle most of the savings you would have gained by using `@extend`, and you can DRY up your stylesheets nicely with mixins.
 
-/* Cell modifiers
-   ========================================================================== */
+### Nested selectors
 
-.cell--detail,
-.cell--important {
-    border-width: 4px;
+**Use no more than 1 level of nesting!**
```

Comment:
I agree, however, there are some exceptions where I think nesting up to 2 levels can be more readable, while still yielding the same code. Example:

```sass
.nav {
    > .item {
        &::after {
            // some styles
        }

        &:last-of-type {
            // some styles
        }
    }
}
```

versus (when maximum nesting level 1 is forced)

```sass
.nav {
    > .item {
        // some styles
    }

    > .item::after {
        // some styles
    }

    > .item:last-of-type {
        // some styles
    }
}
```

Can we somehow accommodate this? What do you guys think @gridonic/core ?